Sources/Shared/Models/PublishJobConnection.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  PublishJobConnection.swift
+//  VimeoNetworking
+//
+//  Copyright © 2019 Vimeo. All rights reserved.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+@objcMembers
+public class PublishJobConnection: VIMConnection {
+    public var publishBlockers: PublishJobBlockers?
+    public var publishConstraints: PublishJobConstraints?
+    
+    public override func getClassForObjectKey(_ key: String?) -> AnyClass? {
+        switch key {
+        case String.Key.publishBlockers:
+            return PublishJobBlockers.self
+        case String.Key.publishConstraints:
+            return PublishJobConstraints.self
+        default:
+            return nil
+        }
+    }
+}
+
+@objcMembers
+public class PublishJobBlockers: VIMModelObject {
+    public var facebook: [String]?
+    public var youtube: [String]?
+    public var linkedin: [String]?
+    public var twitter: [String]?
+}
+
+@objcMembers
+public class PublishConstraints: VIMModelObject {
+    public var duration: NSNumber?
+    public var size: NSNumber?
+}
+
```

Comment:
Should add PublishDestiantions class for map `publish_destinations` object. like
```
public class PublishDestinations: VIMModelObject {
    public var facebook: Bool
    public var youtube: Bool
    public var linkedin: Bool
    public var twitter: Bool
}
```